core/src/test-core/java/com/radixdlt/harness/deterministic/NodeEvents.java

```diff
@@ -100,11 +101,12 @@ public NodeEvents(Map<Class<?>, Set<NodeEventProcessor<?>>> processors) {
     this.processors = Objects.requireNonNull(processors);
   }
 
-  public <T> EventProcessor<T> processor(NodeId node, Class<T> eventClass) {
+  public <T extends CoreEvent> EventProcessor<T> processor(NodeId node, Class<T> eventClass) {
     return t -> processors.get(eventClass).forEach(c -> c.process(node, t));
   }
 
-  public <T> EventProcessorOnDispatch<T> processorOnDispatch(NodeId node, Class<T> eventClass) {
+  public <T extends CoreEvent> EventProcessorOnDispatch<T> processorOnDispatch(
```

Comment:
looks like this whole class is no longer used and can be removed together with (also unused) `SafetyCheckerModule`